test/generate_unittest_netcdfs.py

```diff
@@ -34,9 +34,27 @@ def output_1d_simulated_var_to_netcdf(var_name, file_name, with_bias=False):
     minute = 1 / 60.
     var[:] = [1 * minute, 2 * minute, 3 * minute, 4 * minute]
 
-    var = f.createVariable('MetaData/stationIdentification', str, ('Location'))
-    for i, s in enumerate(["station_1", "station_2", "station_3", "station_4"]):
-        var[i] = s
+    # Station ID
+    if radar_doppler_wind:
+        var = f.createVariable('MetaData/stationIdentification', 'i', ('Location'))
+        var[:] = [1, 2, 3, 4]
```

Comment:
Comment here that radar doppler wind uses integers for `stationIdentification`? And give the reason perhaps?